examples/src/main/python/ml/simple_params_example.py

```diff
@@ -36,18 +35,20 @@
     if len(sys.argv) > 1:
```

Comment:
It seems that the `argv` are never used in this example. So what about just removing this `if` segment?